plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaClientConsumerFactory.java

```diff
@@ -30,32 +29,31 @@
 import static org.apache.kafka.clients.consumer.ConsumerConfig.RECEIVE_BUFFER_CONFIG;
 import static org.apache.kafka.clients.consumer.ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG;
 
-public class PlainTextKafkaConsumerFactory
+public class KafkaClientConsumerFactory
```

Comment:
same as above

Reply:
Yes, this should be fixed after I merge my 2nd commit. KafkaClientsModule.java will set the configs based on SSL or PLAINTEXT.